src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -185,7 +188,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
     )
 
   def cqHead = relationName ~ "(" ~ rep1sep(expr, ",") ~ ")" ^^ {
-    case (r ~ "(" ~ expressions ~ ")") => Atom(r, expressions)
+    case (r ~ _ ~ expressions ~ _) => HeadAtom(r, expressions)
```

Comment:
minor: Does `relationName ~ ("(" ~> rep1sep(expr, ",") <~ ")")` work here, so it can be simply `case (r ~ expressions) =>`?  Just curious.